proposals/NativeInt.md

```diff
@@ -0,0 +1,263 @@
+# Native-sized integers
+
+## Summary
+[summary]: #summary
+
+Language support for a native-sized signed and unsigned integer types.
+
+The motivation is for interop scenarios and for low-level libraries.
+
+## Design
+[design]: #design
+
+New contextual keywords `nint` and `nuint` represent native signed and unsigned integer types.
+The identifiers are only treated as keywords when used in a type context and when the identifier does not otherwise bind to a symbol at that program location.
```

Comment:
>when the identifier does not otherwise bind to a symbol at that program location [](start = 77, length = 80)

This needs to be specified as a modification to the spec for name lookup.  There is no existing spec concept of when an identifier "does not otherwise bind" to a symbol.